packages/template-retail-react-app/app/pages/social-login-redirect/index.jsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {useEffect} from 'react'
+import {FormattedMessage} from 'react-intl'
+import {Box, Container, Stack, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {BrandLogo} from '@salesforce/retail-react-app/app/components/icons'
+import {useCustomerType} from '@salesforce/commerce-sdk-react'
+import useNavigation from '@salesforce/retail-react-app/app/hooks/use-navigation'
+
+const SocialLoginRedirect = () => {
+    const navigate = useNavigation()
+    const {isRegistered} = useCustomerType()
+
+    // If customer is registered push to account page
+    useEffect(() => {
+        navigate('/account')
+    }, [isRegistered])
```

Comment:
This is how the page will be navigated away from. In a following ticket where we wire up the UI to the backend, we will add the call to `loginIDPUser` here on this page as well.